packages/multicast_dns/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: multicast_dns
 description: Dart package for mDNS queries (e.g. Bonjour, Avahi).
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/packages/tree/master/packages/multicast_dns
-version: 0.1.0+1
+version: 0.1.1
```

Comment:
We should update CHANGELOG.md to explain this.

Reply:
ah. forgot that. Done.